javascript-loops-practice/filter-out-nulls.js

```diff
@@ -0,0 +1,10 @@
+/* exported filterOutNulls */
+function filterOutNulls(values) {
+  for (var i = 0; i < values.length; i++) {
+    if (values[i] === null) {
+      values.splice(i, 1);
```

Comment:
Oh no!! You don't want to change the original array, unless it was the specific purpose of the function. Instead of taking values out of the original, you should create a new array with only the values that aren't null